jax_md/quantity.py

```diff
@@ -470,3 +470,15 @@ def update_fn(state: PHopState, position: Array) -> PHopState:
     return PHopState(buff, phop)  # pytype: disable=wrong-arg-count
 
   return init_fn, update_fn
+
+def gamma_from_stokes_law_3d(eta, radii, mass):
```

Comment:
Would it make sense to rename this to be something a little bit more descriptive for someone who doesn't know what gamma is? Perhaps something like `spherical_stokes_friction`? This isn't really my area of expertise, so feel free to pick something else!